soc/arm/st_stm32/stm32h7/sections.ld

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020 Mario Jaun
```

Comment:
@reloZid You're welcome to put your copyright just right down here.

Reply:
Actually this is my copyright ;-) Sorry I'm not very familiar with legal stuff. If I create a new file, who do I have to mention in the copyright? And under which circumstances shall I append my copyright to an existing file?

Reply:
> Actually this is my copyright ;-) Sorry I'm not very familiar with legal stuff. If I create a new file, who do I have to mention in the copyright?

You don't have to mention anybody, if it is your choice. But you can also put just your nickname and date, what should be okay.

> And under which circumstances shall I append my copyright to an existing file?

If I am right, the CI expects copyright. But you can put anything there. Still, some meaningful copyright even with your nickname would be okay.

Reply:
There is no check on this, but I don't think nick names are ok (copyright is "serious stuff")